entiti/user.go

```diff
@@ -0,0 +1,10 @@
+package entiti
```

Comment:
Just typo please fix here and path
```suggestion
package entity
```